### PR TITLE
add custom tos agreed event

### PR DESCRIPTION
**Pull request by goapunk**
add event which indicates that the terms of use have been accepted to other components

### CODE REVIEW

**Review by phillimorland — COMMENTED** [review]
I know this is unfinished but checked it, code looks good but i'm still getting the same behaviour that after f5 refresh the checkbox appears again but thinking it might be my set up?! @khamui can you check this?

**Comment by khamui** [comment]
for me it works, also with refreshing afterwards! 👏

File: adhocracy4/polls/assets/PollQuestions.jsx [hunk 1]
```diff
@@ -38,6 +38,8 @@ class PollQuestions extends React.Component {
       loadingPage: true
     }
 
+    this.handleTermsOfUse = this.handleTermsOfUse.bind(this)
```
Comment (khamui) [inline adhocracy4/polls/assets/PollQuestions.jsx:41]: maybe this can be omitted if you write it as arrow function expression on line 194?

**Review by khamui — COMMENTED** [review]
very cool! working fine

File: adhocracy4/polls/assets/PollQuestions.jsx (hunk 1 above)
Comment (goapunk) [reply adhocracy4/polls/assets/PollQuestions.jsx:41]: hmm, I'd keep it like this till we rewrite it in functional as we do the same in the other components

**Comment by goapunk** [comment]
needs #1120 to be merged first

**Comment by goapunk** [comment]
@phillimorland Just rebased and unwipped as it was depending on the revert of the react update

**Review by phillimorland — APPROVED** [review]
Looking good, linting we can fix when update configs to actually use all our linting packages

**Pull request merged**